docs/source/conf.py

```diff
@@ -77,6 +78,13 @@
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
 
+# Set up RTD GitHub action
+rtds_action_github_repo = "unitaryfund/mitiq"
+rtds_action_path = "mitiq/docs/source/examples"
+rtds_action_artifact_prefix = "notebooks-for-"
+rtds_action_github_token = ${{ secrets.GITHUB_TOKEN }} # noqa: E999,E201,E202
+rtds_action_error_if_missing = False
+
```

Comment:
Could this trick be a solution?  If not, we could still use this trick to set different notebook caching rules for RDT.

```suggestion
import os

# Set up RTD GitHub action
if os.environ.get('READTHEDOCS'):
  rtds_action_github_repo = "unitaryfund/mitiq"
  rtds_action_path = "mitiq/docs/source/examples"
  rtds_action_artifact_prefix = "notebooks-for-"
  rtds_action_github_token = ${{ secrets.GITHUB_TOKEN }} # noqa: E999,E201,E202
  rtds_action_error_if_missing = False
```

Reply:
I think that could work, I'll add and see what happens 🤞